steps/mentor-calibre-fill/configure.yml

```diff
@@ -29,9 +29,10 @@ commands:
   # Merge the design GDS with the dummy fill GDS
   - calibredrv -a layout filemerge \
                -infile [list -name inputs/design.gds -suffix _design] \
-               -infile [list -name dummyfill.gds -suffix _fill] \
-               -createtop {design_name} \
-               -out design_filled.gds \
+               -infile [list -name FEOL.gds -suffix _feol] \
+               -infile [list -name BEOL.gds -suffix _beol] \
```

Comment:
Hm, I wonder if we could make this more generic than FEOL.gds and BEOL.gds... oh well.